sdk/cognitiveservices/cognitiveservices-luis-authoring/src/lUISAuthoringClientContext.ts

```diff
@@ -11,7 +11,7 @@
 import * as msRest from "@azure/ms-rest-js";
 
 const packageName = "@azure/cognitiveservices-luis-authoring";
-const packageVersion = "3.1.0-preview";
+const packageVersion = "4.0.0-preview.1";
```

Comment:
The last published version was `4.0.0-preview.1`. I guess we missed updating this line here the last time.
Assuming you want to continue to be in preview, the next update should be `4.0.0-preview.2`

Reply:
done